test/autotest.py

```diff
@@ -526,6 +526,7 @@ def testRestart():
         cmd+= " "+ckptDir+"/"+i
     #run restart and test if it worked
     procs.append(launch(cmd))
+    sleep(S*SLOW)
```

Comment:
We don't need this sleep, because the `WAITFOR` below already takes care of waiting for 10 seconds. If we indeed want to wait longer, then I would suggest that we change `10` seconds to some higher value if --slow was specified.